datafusion/core/tests/sql/select.rs

```diff
@@ -482,7 +482,7 @@ async fn sort_on_window_null_string() -> Result<()> {
 async fn test_prepare_statement() -> Result<()> {
     let tmp_dir = TempDir::new()?;
     let partition_count = 4;
-    let ctx = partitioned_csv::create_ctx(&tmp_dir, partition_count).await?;
+    let ctx = create_ctx_with_partition(&tmp_dir, partition_count).await?;
```

Comment:
The same code existed in `partitioned_csv.rs` and `mod.rs` - dropped the former and replaced with function calls to `mod.rs`